### PR TITLE
Feature/vagrant build

### PR DESCRIPTION
**Pull request by flibbertigibbet**
Add configuration to optionally build project within vagrant, to be used for testing on Jenkins.

`vagrant up` brings up a VM with the project mounted at `/opt/climate-change-lab` and runs `npm install`.

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+                end
+
+ROOT_VM_DIR = "/opt/climate-change-lab"
+
+Vagrant.configure("2") do |config|
+
+  config.vm.define "app" do |app|
+    app.vm.box = "ubuntu/precise64"
+    app.vm.hostname = "app"
+    app.vm.network "private_network", ip: ENV.fetch("CLIMATE_CHANGE_PRIVATE_IP", "192.168.8.111")
```
Comment (CloudNiner) [inline Vagrantfile]: We should probably add a port forwarding entry here.

**Comment by CloudNiner** [comment]
It would be great to have additional documentation/instructions in the 'Setup' section of the README to note this vm as an additional option for folks who don't want to install node on their host machine.

**Comment by CloudNiner** [comment]
On my mac, after loading up the vm and doing `npm start` from within it, I got this error:
```
ERROR in Missing binding /opt/climate-change-lab/node_modules/node-sass/vendor/linux-x64-48/binding.node
Node Sass could not find a binding for your current environment: Linux 64-bit with Node.js 6.x

Found bindings for the following environments:
  - OS X 64-bit with Node.js 6.x

This usually happens because your environment has changed since running `npm install`.
Run `npm rebuild node-sass` to build the binding for your current environment.
 @ ./src/assets/sass/main.scss 4:14-175 13:2-17:4 14:20-181
```
Doing as the error suggests and running `npm rebuild node-sass` as suggested fixed things up. Could add that command to the provisioner since it wouldn't hurt to run it even if it doesn't need to, and it would save people who are switching OS environments an extra step.

**Comment by flibbertigibbet** [comment]
Didn't see the `node-sass` issue, as I'm on a Linux host. Does that happen when setting up the project for the first time outside of vagrant? If so, that could go in the README.

I'll make the other changes for development usage (was setting it up just for jenkins.)

**Comment by CloudNiner** [comment]
Yeah, I don't think you would see the node-sass issue. It happens if you `npm install` on the host and then `npm install` in the vm, or vice-versa, only if the host and vm are different OSes.

**Comment by CloudNiner** [comment]
+1

**Pull request merged**